tests/unit/amo/components/TestCollection.js

```diff
@@ -879,31 +949,9 @@ describe(__filename, () => {
     const { store } = dispatchSignInActions({
       userProps: { permissions: [COLLECTIONS_EDIT] },
     });
-
-    store.dispatch(
-      loadCurrentCollection({
-        addons: createFakeCollectionAddons(),
-        detail: defaultCollectionDetail,
-      }),
-    );
-
-    const wrapper = renderComponent({ store, _config: fakeConfig });
-
-    const editLink = wrapper.find('.Collection-edit-link').find(Button);
-    expect(editLink).toHaveLength(1);
-    expect(editLink).toHaveProp(
-      'to',
-      `/collections/${defaultUser}/${defaultCollectionDetail.slug}/edit/`,
-    );
-  });
-
-  it('includes the page number in the edit link', () => {
```

Comment:
This test case was removed because what it is testing is also tested in the case for `it('links internally to a Collection edit page')`. We no longer include the text `?page=${page}` in the `pathname`, but rather pass along `filters` in the `query` property of the `to` prop. All of that is being tested in the test case I mention above. There really doesn't seem to be a need for two different tests now. We had 2 before because sometimes the text for `?page=${page}` was included in `pathname` and sometimes it was not, but now we always pass `filters` into the `query` property of the `to`.